sdk/cosmos/azure-cosmos/azure/cosmos/_vector_session_token.py

```diff
@@ -113,6 +113,11 @@ def equals(self, other):
             and self.are_region_progress_equal(other.local_lsn_by_region)
         )
 
+    def is_greater(self, other):
```

Comment:
Not sure why this is a method? Adding methods for what should just be a logic statement is very inefficient in python.